Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.11.1-alpine3.8 as builder
+WORKDIR /go/src/github.com/kubernetes-sigs/aws-ebs-csi-driver
+ADD . .
+RUN CGO_ENABLED=0 GOOS=linux go build -o bin/aws-ebs-csi-driver ./cmd/aws-ebs-csi-driver
+
+FROM registry.fedoraproject.org/fedora-minimal
+COPY --from=builder /go/src/github.com/kubernetes-sigs/aws-ebs-csi-driver/bin/aws-ebs-csi-driver /bin/aws-ebs-csi-driver
+RUN microdnf install -y e2fsprogs && microdnf clean all
+
+ENTRYPOINT ["/bin/aws-ebs-csi-driver", "-logtostderr", "-v", "5"]
```

Comment:
What about moving the arguments (e.g., `-logtostderr`) to the yaml files? This way the user has the option to disable the verbosity.